ansible/roles/package-upgrade/vars/main.yml

```diff
@@ -8,7 +8,7 @@
     'yum': ['centos', 'rhel7', 'aix71', 'aix72', 'ibmi'],
     'apt': ['debian', 'ubuntu'],
     'dnf': ['aix73', 'fedora', 'rhel8'],
-    'pkg': 'freebsd',
+    'pkgng': 'freebsd',
```

Comment:
FWIW I needed to change this to get the Jenkins worker create Ansible playbook to run against our FreeBSD 12 machines otherwise I was getting errors:
```
Could not find a module for pkg.
```
from Ansible (2.11.2).